test/support/nocker.coffee

```diff
@@ -0,0 +1,2 @@
+nock = require('nock')
+nock.disableNetConnect()
```

Comment:
Hopefully this won't be necessary for much longer: https://github.com/pgte/nock/issues/324